drivers/dma/dma_stm32u5.c

```diff
@@ -632,11 +658,12 @@ static int dma_stm32_get_status(const struct device *dev,
 }
 
 static const struct dma_driver_api dma_funcs = {
-	.reload		 = dma_stm32_reload,
-	.config		 = dma_stm32_configure,
-	.start		 = dma_stm32_start,
-	.stop		 = dma_stm32_stop,
-	.get_status	 = dma_stm32_get_status,
+	.reload = dma_stm32_reload,
+	.config = dma_stm32_configure,
+	.start = dma_stm32_start,
+	.stop	= dma_stm32_stop,
+	.get_status	= dma_stm32_get_status,
```

Comment:
Probably a good idea to revert the white space changes so this commit is simply an addition.

Reply:
just adding the new API function